src/pages/events/index.tsx

```diff
@@ -152,12 +152,7 @@ function Table() {
         {/* ACCOUNT AREA*/}
         <div className="absolute right-10 self-center justify-self-end">
           {/* Profile Pop Over */}
-          <ProfilePopover
-            firstName={user?.firstName}
-            lastName={user?.lastName}
-            role={role as UserRole}
-            desc={String(user?.unsafeMetadata.description)}
-          ></ProfilePopover>
+          <ProfilePopover></ProfilePopover>
```

Comment:
If there are no children for this component, then you can write this component in a neater way.

```tsx
<ProfilePopover/>
```

Reply:
hjahhahaha damn, thought i got them all. Missed that one 🧊